src/pages/achievement/dashboard/subcomponents/AchievementModal.tsx

```diff
@@ -0,0 +1,60 @@
+import { Intent } from '@blueprintjs/core';
```

Comment:
Missed this from the first review, but please rename to AchievementView as discussed.